.github/workflows/pull-integration-test.yaml

```diff
@@ -89,6 +96,10 @@ jobs:
           DATABASE_PASSWORD: ${{ steps.int-tests-configs.outputs.DATABASE_PASSWORD }}
           DATABASE_PORT: ${{ steps.int-tests-configs.outputs.DATABASE_PORT }}
           DOCS_TABLE_NAME: ${{ steps.int-tests-configs.outputs.DOCS_TABLE_NAME }}
+          # Langfuse configuration
+          LANGFUSE_HOST:  ${{ steps.int-tests-configs.outputs.LANGFUSE_HOST }}
+          LANGFUSE_PUBLIC_KEY:  ${{ steps.int-tests-configs.outputs.LANGFUSE_PUBLIC_KEY }}
+          LANGFUSE_SECRET_KEY:  ${{ steps.int-tests-configs.outputs.LANGFUSE_SECRET_KEY }}
         run: |
           echo "${{ secrets.MODELS_JSON }}" | base64 --decode | jq > $GITHUB_WORKSPACE/config/config.json
```

Comment:
I saw it later that we can add the Langfuse config in MODELS_JSON config. Let me add inside the it and then we don't need any change.